### PR TITLE
Add Http servlet listener and filter.

### PR DESCRIPTION
**Pull request by rghetia**

### CODE REVIEW

**Comment by rghetia** [comment]
fixes #1643

File: contrib/http_servlet/build.gradle [hunk 1]
```diff
@@ -10,12 +10,14 @@ description = 'OpenCensus Http Servlet Plugin'
 }
 
 def opencensusVersion = "0.18.0-SNAPSHOT"
```
Comment (bogdandrutu) [inline contrib/http_servlet/build.gradle]: shouldn't this be 0.19.0-SNAPSHOT?

Comment (bogdandrutu) [reply contrib/http_servlet/build.gradle]: I think you should do the same as you did for http util and change every 0.18.0 to 0.19.0 (including since)

Comment (rghetia) [reply contrib/http_servlet/build.gradle]: the above version is not used. I'll open another PR and remove it. There are no other since 0.18.0

Comment (rghetia) [reply contrib/http_servlet/build.gradle]: there is however reference to 0.18.0 in readme. I'll fix it.

**Pull request merged**